components/automate-ui/src/app/pages/notifications/notifications.component.ts

```diff
@@ -39,13 +46,18 @@ export class NotificationsComponent implements OnInit {
   permissionDenied = false; // not currently used
   // This is exposed here to allow the component HTML access to ServiceActionType
   serviceActionType = ServiceActionType;
+
   public notificationToDelete: NotificationRule;
   public deleteModalVisible = false;
+  public hookStatus = UrlTestState.Inactive;
+  public notificationObj = new NotificationRule('', '', null, '', null, '', false);
+  private isDestroyed = new Subject<boolean>();
+
+  public openNotificationModal = new EventEmitter<boolean>();
```

Comment:
I realize you adapted the existing model code for this (👍 ), but I think we can improve it slightly. We do not need a boolean at all; we just need the event.
Make this a `void` (or `null` also would work here):
```suggestion
  public openNotificationModal = new EventEmitter<void>();
```
Then change the emit below from this
```
this.openNotificationModal.emit(true);
```
to this
```
this.openNotificationModal.emit();
```
And similarly need a change in create-notification-modal.component.ts from `<boolean>` to `<void>`.